src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -69,6 +70,17 @@ public final class ThrowableToErrorMapper {
           return handleDriverException((DriverException) throwable, message);
         }
 
+        // handle an invalid Content-Type header
+        if (throwable instanceof NotSupportedException
+            && throwable
+                .getMessage()
+                .contains("The content-type header value did not match the value")) {
+          // validate the Content-Type header, 415 if failed
+          return ErrorCode.INVALID_CONTENT_TYPE_HEADER
+              .toApiException()
+              .getCommandResultError(Response.Status.UNSUPPORTED_MEDIA_TYPE);
```

Comment:
It's too bad actual incorrect Content-Type seems not available from `NotSupportedException` -- if it was, would be good to include in exception message.

Reply:
Yeah, that is not included, unfortunately.